javascript/javascript3/week2/CurrencyCalculator/app.js

```diff
@@ -0,0 +1,45 @@
+const fromRate = document.getElementById('from')
+const toRate = document.getElementById('to')
+const amount = document.getElementById('amount')
+const btn = document.getElementById('btn')
+const result = document.getElementById('result')
+const getData = async() => {
+    try {
+        const response = await fetch("https://open.er-api.com/v6/latest/USD");
+        const currencyData = await response.json();
+        const currencyRate = Object.keys(currencyData.rates)
+        currencyRate.map((ele) => {
+            if (ele === 'EUR') {
+                fromRate.innerHTML += `<option selected="selected" value="${ele}">${ele}</option>`
+            } else {
+                fromRate.innerHTML += `<option value="${ele}">${ele}</option>`
+            }
+            if (ele === 'DKK') {
+                toRate.innerHTML += `<option selected="selected" value="${ele}">${ele}</option>`
+            } else {
+                toRate.innerHTML += `<option value="${ele}">${ele}</option>`
+            }
+        })
+    } catch (err) {
+        console.log(err.message);
+    }
+}
+getData()
+
+const calcData = async() => {
+    try {
+        const convertToValue = toRate.value
+        const convertFromValue = fromRate.value
+        const response = await fetch(`https://open.er-api.com/v6/latest/${fromRate.value}`);
+        const data = await response.json();
+        console.log(data.rates)
```

Comment:
Remove console.log before submitting homework (it is ok to leave it with error messages).